tests/release/e2e-test-push-image-to-pyxis.go

```diff
@@ -153,62 +170,116 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-1571]test-release-e2e-push-image-
 				if err != nil || prList == nil || len(prList.Items) < 1 {
 					return false
 				}
+				componentHasBuildPr := false
+				additionalComponentHasBuildPr := false
+				for _, pr := range prList.Items {
+					if strings.Contains(pr.Name, componentName) {
+						componentHasBuildPr = true
+						buildPrName = pr.Name
+					} else if strings.Contains(pr.Name, additionalComponentName) {
+						additionalComponentHasBuildPr = true
+						additionalBuildPrName = pr.Name
+					}
+				}
+
+				GinkgoWriter.Printf("\nFirst Build PirpelineRun:\n %s", buildPrName)
+				GinkgoWriter.Printf("\nSecond Build PirpelineRun:\n %s", additionalBuildPrName)
 
-				return strings.Contains(prList.Items[0].Name, componentName)
+				return componentHasBuildPr && additionalComponentHasBuildPr
 			}, releasePipelineRunCreationTimeout, defaultInterval).Should(BeTrue())
 		})
 
-		It("verifies that the PipelineRun in dev namespace succeeded.", func() {
+		It("verifies that the build PipelineRun in dev namespace succeeded.", func() {
 			Eventually(func() bool {
-				prList, err := fw.AsKubeAdmin.TektonController.ListAllPipelineRuns(devNamespace)
-				if prList == nil || err != nil || len(prList.Items) < 1 {
+				buildPr, err := fw.AsKubeAdmin.TektonController.GetPipelineRun(buildPrName, devNamespace)
+				if err != nil {
+					GinkgoWriter.Printf("\nError getting PirpelineRun %s:\n %s", buildPrName, err)
+					return false
+				}
+				additionalBuildPr, err := fw.AsKubeAdmin.TektonController.GetPipelineRun(additionalBuildPrName, devNamespace)
+				if err != nil || additionalBuildPr == nil {
```

Comment:
I'm not sure if the `additionalBuildPr == nil` part is needed - where would it disappear (since we found it in the previous block)? I don't mind having it, but in that case we should have the same for the first one (`buildPr`) for consistency.

Reply:
agree with you, 
I'll remove the  `additionalBuildPr == nil`

Reply:
Cool, thanks.